components/Footer.jsx

```diff
@@ -1,47 +1,72 @@
-import { FaGithub, FaTwitter, FaDiscord, FaLinkedin } from "react-icons/fa";
-const Footer = () => {
+import React from 'react';
+import { FaDiscord, FaGithub, FaLinkedin, FaTwitter } from 'react-icons/fa';
+
+function Footer () {
   return (
-    <footer className="md:max-md: flex flex-col items-center gap-4 bg-[#0d1117] p-4 py-8 text-center text-white/80">
+    <footer className="md:max-md flex flex-col items-center gap-4 bg-[#0d1117] p-4 py-8 text-center text-white/80">
       <div className="flex items-center">
         <div className="flex gap-10">
           <a
-            target="_blank"
             className="text-2xl hover:text-white"
             href="https://github.com/FrancescoXX/4c-site"
+            rel="noreferrer"
+            target="_blank"
           >
-            <span className="sr-only">Github Repository</span>
+            <span className="sr-only">
+              Github Repository
+            </span>
+
             <FaGithub aria-hidden="true" />
           </a>
+
           <a
-            target="_blank"
             className="text-2xl hover:text-white"
             href="https://discord.com/invite/TcmA2kbJeA"
+            rel="noreferrer"
+            target="_blank"
           >
-            <span className="sr-only">Discord Server</span>
+            <span className="sr-only">
+              Discord Server
+            </span>
+
             <FaDiscord aria-hidden="true" />
           </a>
+
           <a
-            target="_blank"
             className="text-2xl hover:text-white"
             href="https://twitter.com/4ccommunityhq"
+            rel="noreferrer"
+            target="_blank"
           >
-            <span className="sr-only">Twitter</span>
+            <span className="sr-only">
+              Twitter
+            </span>
+
             <FaTwitter aria-hidden="true" />
           </a>
+
           <a
-            target="_blank"
             className="text-2xl hover:text-white"
             href="https://www.linkedin.com/company/4ccommunity/about/"
+            rel="noreferrer"
```

Comment:
I've just reviewed this and I'm only going to let the noreferrer thing slide for now, because these links are links to social media platforms. In the future when the official 4c site is working with 3rd-party partners, we won't be needing the "noreferrer" value

Reply:
Talked about in Discord. Also I don't think I really added this change